teachopencadd/talktorials/019_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1250 @@
+{

```

Comment:
<p>I thought OpenForceField could convert to OpenMM just fine. Am I missing something?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>Since we have to convert the unit of the positions we extract topology and positions separately and create a modeller object from there. Maybe a modeller object could be created from the openff Molecule directly, but what about the unit conversion then? We could probably do it at another step of the process, if that's better. Any ideas? </p><p><a href="https://open-forcefield-toolkit.readthedocs.io/en/latest/api/generated/openff.toolkit.topology.Molecule.html#openff.toolkit.topology.Molecule" target="_blank">OpenFF Molecule Docs</a></p>